testng-core/src/main/java/org/testng/internal/Parameters.java

```diff
@@ -416,14 +417,26 @@ private static void checkParameterTypes(
           annotation = "@" + methodAnnotation;
         }
         String errPrefix;
+        boolean nativeInjectionUnsupported =
```

Comment:
It is no need to declare it here as it is only used into the if block. If you declare later we will save some instructions.

Reply:
Fixed. Please check now.